### PR TITLE
feat: add ci and container build

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,39 @@
 ## Introduction
 
 This container should be used to deploy code with r10k. It has the r10k gem and all dependencies installed.
+
+## Version Schema
+
+The version schema has the following layout:
+
+```text
+<puppet.major>.<puppet.minor>.<puppet.patch>-v<container.major>.<container.minor>.<container.patch>
```
Comment (bastelfreak) [inline README.md]: puppet.major?

Comment (rwaffen) [reply README.md]: copy error... i copied it from voxbox 😅

File: build_versions.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "include": [
+    {
+      "rubygem_r10k": "4.1.0",
```
Comment (bastelfreak) [inline build_versions.json]: we did the 5.0.0 release yesterday, do you want to update?

Comment (rwaffen) [reply build_versions.json]: sounds good, will do

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**